exercises/rest-api/RestApi.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+public class User
```

Comment:
Could you remove this class? It is not necessary for the public test API and a user should be free to choose the underlying representation he/she chooses.